packages/strapi-plugin-users-permissions/admin/src/translations/de.json

```diff
@@ -22,32 +26,43 @@
   "List.title.providers.enabled.singular": "{number} Methode ist aktiviert und",
   "Plugin.permissions.plugins.description": "Definiere die möglichen Aktionen des {name} Plugins.",
   "Plugins.header.description": "Nur Aktionen, die an einen Pfad gebunden sind, werden hier gelistet.",
-  "Plugins.header.title": "Befugnisse",
+  "Plugins.header.title": "Berechtigungen",
   "Policies.InputSelect.empty": "Keine",
   "Policies.InputSelect.label": "Diese Aktion folgenden erlauben:",
   "Policies.header.hint": "Wähle eine Aktion aus und klicke auf das Zahnrad, um den an diese Aktion gebundenen Pfand anzuzeigen",
   "Policies.header.title": "Fortgeschrittene Einstellungen",
-  "PopUpForm.Email.email_templates.inputDescription": "Für mehr Informationen, {link}",
+  "PopUpForm.Email.email_templates.inputDescription": "{link} für mehr Informationen",
+  "PopUpForm.Email.link.documentation": "Lies die Dokumentation",
   "PopUpForm.Email.options.from.email.label": "E-Mail-Adresse des Absenders",
-  "PopUpForm.Email.options.from.email.placeholder": "kai@doe.com",
+  "PopUpForm.Email.options.from.email.placeholder": "max@mustermann.de",
   "PopUpForm.Email.options.from.name.label": "Name des Absenders",
-  "PopUpForm.Email.options.from.name.placeholder": "Kai Doe",
+  "PopUpForm.Email.options.from.name.placeholder": "Max Mustermann",
```

Comment:
Should I just replace the mail adress or also the name right here?

Reply:
my bad the name too :)